proposal.rst

```diff
@@ -1242,6 +1242,23 @@ components, thus freeing both server and application authors from
 implementing the same features over and over again.
 
 
+HTTP/2 Features
+---------------
+
+From the perspective of WSGI applications and servers, HTTP/2 is not handled
+differently from HTTP/1.1. Because the semantics of HTTP/2 requests and
+responses are identical to the semantics of HTTP/1.1 requests and responses,
+there is no requirement for servers to treat them differently in WSGI.
+
+Servers that support both HTTP/1.1 and HTTP/2 **may** provide support for
```

Comment:
i'm not sure how the "preload" spec is related to the server... The server has not access to the "link" information, it's rather an hint for the client that may be sent via the link header or not to the server. However, we need to introduce a way to tell to the application it can place such events on the server that support it. 

Something like

```python
environ['wsgi.preload'].push(id, Stream)
```

where stream would be a fileobject like. 

Thoughts?